spyder/utils/introspection/utils.py

```diff
@@ -211,6 +211,30 @@ def get_keywords(lexer):
     return keywords
 
 
+def get_words_file(file_in):
```

Comment:
This methods will be called most likely using the content (that is already available for an editor) and not the file name, so the signature should be more like

```python
def get_words_file(file_path=None, content=None):
    if file_path is None and content is None or file_path and content:
        error_msg = _('Must provide one of `file_path` or `content`')
```